app/jobs/clock/webhooks_cleanup_job.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Clock
+  class WebhooksCleanupJob < ApplicationJob
+    queue_as 'clock'
+
+    def perform
+      Webhook.where('updated_at < ?', 90.days.ago).destroy_all
```

Comment:
Do we really need to hard-destroy them? 🤔 

Maybe hiding them by an archived status?

Reply:
I think we don't need to keep them, it will possibly be a very large table if we keep everything